examples/tutorial.py

```diff
@@ -929,13 +1038,402 @@ def get_overlapping_labels(array1, array2):
+    process_function=lambda b: ...,  # Empty process function
+    fit="overhang",
+)
+valid_task = daisy.Task(
+    "valid",  # We give the task a name
+    total_roi=Roi((0,), (100,)),  # a 1-D bounding box [0,100)
+    read_roi=Roi((0,), (9,)),  # We read in blocks of size [0,1)
+    write_roi=Roi((2,), (5,)),  # We write in blocks of size [0, 1)
+    process_function=lambda b: ...,  # Empty process function
+    fit="valid",
+)
+
+# %% [markdown]
+# because we now have:
+#
+# total roi: [0, 100)
+#
+# read roi: [0, 9)
+#
+# write roi: [2, 7)
+#
+# There is a "context" of 2 pixels between the read_roi and write roi.
+# Removing this context from the total roi gives us a "total write roi" of [2, 98).
+# Note that the total number of pixels to tile is 96, which is not evenly divisible
+# by the chunk size of 5. So our final block will have write roi of [97, 102) and a
+# read roi of [95, 104). Depending on your task, reading, and especially writing outside
+# of the given total roi bounds can cause problems.
+# in these cases we can handle the final block in a few ways. "valid": ignore the
+# final block. "overhang": process the final block. "shrink": shrink the read/write
+# rois until the read roi is fully contained in the total roi. In this case the "shrink"
+# strategy would shrink the read roi to [95, 100) and the write roi to [97, 98).
+#
+# Now when we executing the overhang task it will have one more block processed than the
+# valid task
+
+# %%
+daisy.run_blockwise([overhang_task], multiprocessing=False)
+daisy.run_blockwise([valid_task], multiprocessing=False)
+
+# %% [markdown]
+# ### Task Chaining
+#
+# It is also possible to chain multiple tasks together in such a way that
+# they either run in parallel or in sequence. If running in parallel, we
+# simply start the workers for both tasks at once. If running in sequence,
+# blocks will only be realeased once their dependencies in previous tasks
+# have been completed.
+
+# %%
+total_roi = Roi((0, 0), (1250, 1250))
+read_roi_shape = (400, 400)
+write_roi_shape = (250, 250)
+context = Coordinate(75, 75)
+
+task_a_to_b = daisy.Task(
+    "A_to_B",
+    total_roi=total_roi.grow(
+        context,
+        context,
+    ),
+    read_roi=Roi((0, 0), read_roi_shape),
+    write_roi=Roi(context, write_roi_shape),
+    process_function=lambda b: ...,
+    read_write_conflict=False,
+)
+task_b_to_c = daisy.Task(
+    "B_to_C",
+    total_roi=total_roi.grow(
+        context,
+        context,
+    ),
+    read_roi=Roi((0, 0), read_roi_shape),
+    write_roi=Roi(context, write_roi_shape),
+    process_function=lambda b: ...,
+    read_write_conflict=False,
+    upstream_tasks=[task_a_to_b],
+)
+
+daisy.run_blockwise([task_a_to_b, task_b_to_c], multiprocessing=False)
+
+# %% [markdown]
+#
+# Here is an mp4 visualization of the above task
+#
+# ![task chaining](_static/task_chaining.gif)
+
+# %% [markdown]
+# # Map Reduce and Some simple benchmarks
```

Comment:
I don't really understand what information this part is trying to impart - I'd probably remove it

Reply:
I think the main value seemed to be the timings. I figured it can't hurt to have a few more simple examples